settings/src/components/webauthn/webauthn.js

```diff
@@ -66,42 +48,45 @@ export default function WebAuthn() {
 				path: '/wporg-two-factor/1.0/provider-status',
 				method: 'POST',
 				data: {
-					user_id: record.id,
+					user_id: userId,
 					provider: 'TwoFactor_Provider_WebAuthn',
-					status: 'enable',
+					status: newStatus,
 				},
 			} );
 
 			await refreshRecord( userRecord );
-			setGlobalNotice( 'Successfully enabled Security Keys.' );
+			setGlobalNotice( `Successfully ${ newStatus }d Security Keys.` );
 		} catch ( error ) {
 			setStatusError( error?.message || error?.responseJSON?.data || error );
```

Comment:
I got a pretty nerdy error when i cancelled out of creating the key. Not sure if this was the source line but here is the message:

<img width="769" alt="Screenshot 2023-08-11 at 3 44 22 PM" src="https://github.com/WordPress/wporg-two-factor/assets/1657336/a7ebca7c-b2c7-4942-a18c-9041ae504aed">

Reply:
The content of the error is coming from the browser, so we don't have direct control over it. In this case, [it's intentionally vague for privacy](https://www.w3.org/TR/webauthn-2/#sctn-privacy-considerations-client).

IIRC, our app intentionally shows errors to the user, since that makes troubleshooting much easier. They can tell us exactly what went wrong, rather than only having a vague error. I don't feel strongly about that, though.

Reply:
I'm guessing this is the source:

https://github.com/WordPress/wporg-two-factor/blob/5cd6e01b413a2e3c3f590661126e0d901ed881fb/settings/src/components/webauthn/register-key.js#L51-L63

Reply:
> IIRC, our app intentionally shows errors to the user, since that makes troubleshooting much easier. They can tell us exactly what went wrong, rather than only having a vague error. I don't feel strongly about that, though

I agree with that policy. In this case, I've taken a specific, purposeful action to cancel. I wouldn't expect that to trigger an error of any kind.

Reply:
I've added a [commit](https://github.com/WordPress/wporg-two-factor/pull/223/commits/cfe15185c9f792fc2ee5e359123af28293214784) to ignore this particular exception, unsure how many scenarios might be covered by that

Reply:
👍🏻